outputs.tf

```
@@ -1,10 +1,10 @@
 output "log_group_arn" {
-  value       = "${aws_cloudwatch_log_group.default.arn}"
+  value       = "${aws_cloudwatch_log_group.default.*.arn}"
```

```suggestion
  value       = "${join("", aws_cloudwatch_log_group.default.*.arn)}"
```

Use `splat+join` pattern for resources with `count` since they are lists